project/plugin.sbt

```diff
@@ -15,19 +15,21 @@ def printAndDie(msg: String): Nothing = {
 // Update SCALAJS_VERSION in release.sh, as well
 val scalaJSVersion = env("SCALAJS_VERSION") match {
   case Some("0.6.33") => "0.6.33"
-  case Some("1.3.0") | None => "1.3.0"
+  case Some("1.3.0") | Some("") | None => "1.3.0"
```

Comment:
Gosh, Erik wanted to have the build script be fatal if the wrong version was supplied, but I suppose that never worked.

Maybe we should delete the `printAndDie`, below, since it doesn't do anything.

Reply:
Well, if the wrong version is specified, it *is* fatal. It's just that some platforms don't distinguish between an empty envar and an unspecified one.

Reply:
Yes, I misread it.  You're right, it's fine.